tests/test_lint_recipe.py

```diff
@@ -3313,23 +3333,38 @@ def test_hint_pip_no_build_backend(
         ),
     ],
 )
+@pytest.mark.parametrize("skip", [False, True])
 def test_hint_noarch_python_use_python_min(
     meta_str,
     expected_hints,
+    skip,
+    tmp_path,
 ):
+    if skip:
+        if not expected_hints:
+            pytest.skip("No hints expected")
```

Comment:
Why should we need to skip this test if there are no expected hints? We could reformulate the test below to expect empty hints, no?

Reply:
It always expected empty hints. I've skipped it because it's redundant, i.e. doesn't test anything.

Reply:
That makes sense, I'll update it. In the end, the skips don't save that much time, and could be confusing.